component-test/test_dbus_proxy.py

```diff
@@ -155,6 +155,152 @@
     "opath_1": stubs.OPATH_1
 })
 
+CONF_WHITELIST_DIRECTIONS = """
+{{
+    "dbus-gateway-config-session": [{{
+        "direction": "*",
+        "interface": "*",
+        "object-path": "*",
+        "method": "*"
+    }},
+    {{
+        "direction": "outgoing",
+        "interface": "{iface}.{extension_1}",
+        "object-path": "*",
+        "method": "*"
+    }}],
+    "dbus-gateway-config-system": []
+}}
+""".format(**{
+    "iface": stubs.IFACE_1,
+    "extension_1": stubs.EXT_1
+})
+
+CONF_WHITELIST_DIRECTIONS_REVERSE = """
+{{
+    "dbus-gateway-config-session": [{{
+        "direction": "outgoing",
+        "interface": "{iface}.{extension_1}",
+        "object-path": "*",
+        "method": "*"
+    }},
+    {{
+        "direction": "*",
+        "interface": "*",
+        "object-path": "*",
+        "method": "*"
+    }}],
+    "dbus-gateway-config-system": []
+}}
+""".format(**{
+    "iface": stubs.IFACE_1,
+    "extension_1": stubs.EXT_1
+})
+
+CONF_WHITELIST_PATH = """
+{{
+    "dbus-gateway-config-session": [{{
+        "direction": "*",
+        "interface": "*",
+        "object-path": "*",
+        "method": "*"
+    }},
+    {{
+        "direction": "incoming",
+        "interface": "{iface}.{extension_1}",
+        "object-path": "/a/path/to/unavailable/directory/",
+        "method": "*"
+    }}],
+    "dbus-gateway-config-system": []
+}}
+""".format(**{
+    "iface": stubs.IFACE_1,
+    "extension_1": stubs.EXT_1
+})
+
+CONF_WHITELIST_PATH_REVERSE = """
```

Comment:
There are four configuration that are grouped by two. And the first section is needed for both white-listing test for direction and also to show direction is priorities among others. The second one is just for creating an white-listing test for path when all chains can be considered as valid configuration. Thats why these 2 section is seperated.